integration-test/src/main/java/com/sequenceiq/it/cloudbreak/assertion/distrox/AvailabilityZoneAssertion.java

```diff
@@ -0,0 +1,66 @@
+package com.sequenceiq.it.cloudbreak.assertion.distrox;
+
+import java.util.HashMap;
+import java.util.List;
+import java.util.Map;
+import java.util.Set;
+
+import org.jetbrains.annotations.NotNull;
+
+import com.sequenceiq.cloudbreak.api.endpoint.v4.stacks.response.instancegroup.InstanceGroupV4Response;
+import com.sequenceiq.cloudbreak.api.endpoint.v4.stacks.response.instancegroup.instancemetadata.InstanceMetaDataV4Response;
+import com.sequenceiq.cloudbreak.common.mappable.CloudPlatform;
+import com.sequenceiq.it.cloudbreak.CloudbreakClient;
+import com.sequenceiq.it.cloudbreak.assertion.Assertion;
+import com.sequenceiq.it.cloudbreak.context.TestContext;
+import com.sequenceiq.it.cloudbreak.dto.distrox.DistroXTestDto;
+import com.sequenceiq.it.cloudbreak.exception.TestFailException;
+
+public class AvailabilityZoneAssertion implements Assertion<DistroXTestDto, CloudbreakClient> {
+
+    @Override
+    public DistroXTestDto doAssertion(TestContext testContext, DistroXTestDto testDto, CloudbreakClient client) throws Exception {
+        if (testDto.getCloudPlatform() != CloudPlatform.AWS) {
```

Comment:
do we need to also check the variant?

Reply:
i haven't seen it in the response.. I chose to check the metadata availibitily zone size instead